tests/e2e_tests/orchestration/test_e2e_output_builder.py

```diff
@@ -66,6 +66,6 @@ def test_e2e_build_debug_output(
     mock_iris_pipeline_call_trace: PipelineCallTraceStorage,
```

Comment:
We'll have to add an e2e test for the simple debugging case too - bot blocking for this PR